ROADMAP-TO-1.0.0.md

```diff
@@ -156,16 +109,15 @@ These are some libraries that must exist to make developers' lives easy. It is c
   - [x] ipfs api bindings in Java
   - [x] ipfs api bindings in Python
   - [x] ipfs api bindings in PHP
-  - [ ] ipfs api bindings in Ruby (in progress)
-  - [ ] ipfs api bindings in Scala (in progress)
-  - [ ] ipfs api bindings in Rust (in progress)
-  - [ ] ipfs api bindings in Haskell
-  - [ ] ipfs api bindings in C/C++
+  - [x] ipfs api bindings in Ruby
+  - [x] ipfs api bindings in Scala
+  - [x] ipfs api bindings in Rust
+  - [x] ipfs api bindings in Haskell
+  - [x] ipfs api bindings in C/C++
   - [ ] ipfs api bindings in Erlang
 - [x] full ipfs node implementations
   - [x] go-ipfs - implementation in Go
-  - [ ] js-ipfs - implementation in JS node
-  - [ ] js-ipfs-browser - implementation in JS browser
+  - [x] js-ipfs - implementation in JS
```

Comment:
There should be something about full interop between the implementations, as it stand to say `js-ipfs` is done is not really the case imho. It exists, but still missing a lot to be ready for a 1.0

Reply:
Which is not to say, it's not in a pretty good state, but I don't think we are considering even go-ipfs 1.0 ready at this point, so I am wondering what the `[x]` really means in this list as `js-ipfs` or `go-ipfs` are not really measurable use cases if you compare it to sth like `Scales to 10,000 nodes`.